src/commands/index.ts

```diff
@@ -1,21 +1,10 @@
 import * as vscode from 'vscode';
 import { getConfig } from '../utils/vscode-extensions';
 import { LanguageResourceManager } from '../i18n';
-import {
-  IconGenerator,
-  mergeConfig,
-  schema,
-  toggleAngularPreset,
-  toggleOfficialIconsPreset,
-  toggleHideFoldersPreset,
-} from '../icon-manifest';
+import * as iconManifest from '../icon-manifest';
```

Comment:
Why did you decide to make a generic import at the cost of making the code more verbose?